contracts/smart-account/interfaces/modules/ISecurityPolicyPlugin.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+/// @title Security Policy Plugin
+/// @author @ankurdubey521
+/// @dev A security policy plugin is a plugin which enforce arbitrary checks and condition during plugin installation on the smart contract wallet
```

Comment:
`/// @dev A security policy plugin enforces checks during plugin installation on the smart contract wallet.` so it can fits on one line